adapters/bluesea/bluesea.go

```diff
@@ -0,0 +1,148 @@
+package bluesea
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	"github.com/prebid/openrtb/v17/openrtb2"
```

Comment:
We upgraded the version of the openrtb library from v17 to v19. Please update your import statements to use `github.com/prebid/openrtb/v19/openrtb2`.

Reply:
hi @Sonali-More-Xandr   the openrtb lib has been updated.